src/background/services/background.ts

```diff
@@ -90,6 +118,10 @@ export class Background {
 
             case PopupToBackgroundAction.CONNECT_WALLET:
               await this.openPaymentsService.connectWallet(message.payload)
+              await this.browser.alarms.clear(ALARM_RESET_OUT_OF_FUNDS)
```

Comment:
Since `CONNECT_WALLET` is only called during setup, there is no point in clearing the alarm here. We can clear the alarm on `DISCONNECT_WALLET`.